README.md

```diff
@@ -35,3 +35,58 @@
  * Star this repo, if you like the project. 
  * Request a new feature by [opening a issue](https://github.com/bk1285/rpi_wordclock/issues), describing your feature request.
  * Contribute your code: [Learn how to create your own plugin](https://rpi-wordclock.readthedocs.io/en/master/doc_further_reading.html#adding-a-new-plugin)
+
+### Python3
```

Comment:
Fine for now, while I see this in the long run spilt up into the documentation (readthedocs) and maybe a changelog (esp. the remarks on flask-restplus).